src/gretel_trainer/relational/multi_table.py

```diff
@@ -557,7 +558,73 @@ def classify(self, config: GretelModelConfig, all_rows: bool = False) -> None:
         )
 
     def train_transform_models(self, configs: Dict[str, GretelModelConfig]) -> None:
+        """
+        DEPRECATED: Please use `train_transforms` instead.
+        """
+        logger.warning(
+            "This method is deprecated and will be removed in a future release. "
+            "Please use `train_transforms` instead."
+        )
+        use_configs = {}
         for table, config in configs.items():
+            for m_table in self.relational_data.get_modelable_table_names(table):
+                use_configs[m_table] = config
+
+        for table, config in use_configs.items():
+            transform_config = make_transform_config(
+                self.relational_data, table, config
+            )
+
+            # Ensure consistent, friendly data source names in Console
+            table_data = self.relational_data.get_table_data(table)
+            transforms_train_path = self._working_dir / f"transforms_train_{table}.csv"
+            table_data.to_csv(transforms_train_path, index=False)
+
+            # Create model
+            model = self._project.create_model_obj(
+                model_config=transform_config, data_source=str(transforms_train_path)
+            )
+            self._transforms_train.models[table] = model
+
+        self._backup()
+
+        task = TransformsTrainTask(
+            transforms_train=self._transforms_train,
+            multitable=self,
+        )
+        run_task(task, self._extended_sdk)
```

Comment:
This is copy/paste of the original method—I think I'll consolidate it into a single method. Also working on getting some unit tests around this, will push both to this branch.